deploy/helm_charts/envs/worldbank.yaml

```diff
@@ -32,7 +32,7 @@ mixer:
   useBaseBigtable: false
   useBranchBigtable: false
   useCustomBigtable: true
-  remoteMixerDomain: https://api.datacommons.org
+  remoteMixerDomain: https://autopush.api.datacommons.org
```

Comment:
nit: Consider adding a note to update this URL before launch.

Reply:
Done